bin/stub-server.js

```diff
@@ -4,11 +4,16 @@
 
 /* eslint-disable @typescript-eslint/no-var-requires */
 
+const cmd = require('commander');
 const express = require('express');
 const { resolve } = require('path');
 
 const { stubServer } = require('../dist/cjs/stubServer');
 
+cmd.option('-p, --port <port>', 'stub server port').option('-c, --config <config>', 'config file');
```

Comment:
Defaults should be documented/set here.
See https://github.com/tj/commander.js/#default-option-value